src/components/cesium_map/api/spatial.js

```diff
@@ -165,13 +165,17 @@ export class PointStreamPrimitiveCollection extends Cesium.PointPrimitiveCollect
           } else {
             locations[location] = 1;
           }
+          if (Array.isArray(doc.z)){
+            doc.z = doc.z[0] // get integer value if multivalue field
+          }
```

Comment:
We might not need this checking after re-indexing hyde, but included it for now.